tests/suite/math/call.typ

```diff
@@ -8,6 +8,97 @@ $ pi(a,) $
 $ pi(a,b) $
 $ pi(a,b,) $
 
+--- math-call-unclosed-func ---
+#let func(x) = x
+// Error: 1-2 unclosed delimiter
+// Error: 6-7 unclosed delimiter
+// Error: 8-9 unexpected dollar sign
+$func(a$
+
+--- math-call-unclosed-non-func ---
+// Error: 1-2 unclosed delimiter
+// Error: 5-6 unclosed delimiter
+// Error: 7-8 unexpected dollar sign
+$sin(x$
+
+--- math-call-named-args ---
+#let func1(my: none) = my
+#let func2(_my: none) = _my
+#let func3(my-body: none) = my-body
+#let func4(_my-body: none) = _my-body
+#let func5(m: none) = m
+$ func1(my: a) $
+$ func2(_my: a) $
+$ func3(my-body: a) $
+$ func4(_my-body: a) $
+$ func5(m: a) $
+$ func5(m: sigma : f) $
+$ func5(m: sigma:pi) $
+
+--- math-call-named-args-duplicate ---
+#let func(my: none) = my
+// Error: 15-17 duplicate argument: my
+$ func(my: a, my: b) $
+
+--- math-call-named-single-underscore ---
+#let func(x) = x
+// Error: 8-9 expected identifier, got underscore
+$ func(_: a) $
+
+--- math-call-named-single-char-error ---
```

Comment:
The name is a bit odd. The problem seems to be the space after the "m" more than the fact that it's a single char.

Reply:
My point was that if it weren't a single character you'd get an unknown variable error instead.